webui/server/server.py

```diff
@@ -151,7 +151,8 @@ def __init__(self, profile_handler, port='', timeout=1):
     self.ser = None
     self.port = port
     self.timeout = timeout
-    self.write_queue = queue.Queue(10)
+    # Sized for `num_panels` threshold updates plus one "v" command
+    self.write_queue = queue.Queue(num_panels + 1)
```

Comment:
Also while we're here, can we rename `num_panels` to `num_sensors` so that it's more accurate and no longer a misnomer.